markets/perps-market/contracts/modules/LiquidationAssetManagerModule.sol

```diff
@@ -0,0 +1,101 @@
+//SPDX-License-Identifier: MIT
+pragma solidity >=0.8.11 <0.9.0;
+
+import {OwnableStorage} from "@synthetixio/core-contracts/contracts/ownership/OwnableStorage.sol";
+import {GlobalPerpsMarketConfiguration} from "../storage/GlobalPerpsMarketConfiguration.sol";
+
+import {LiquidationAssetManager} from "../storage/LiquidationAssetManager.sol";
+import {AddressUtil} from "@synthetixio/core-contracts/contracts/utils/AddressUtil.sol";
+import {AddressError} from "@synthetixio/core-contracts/contracts/errors/AddressError.sol";
+import {ParameterError} from "@synthetixio/core-contracts/contracts/errors/ParameterError.sol";
+import {Clones} from "../utils/Clones.sol";
+import {IPerpRewardDistributor} from "@synthetixio/perps-reward-distributor/contracts/interfaces/IPerpsRewardDistributor.sol";
+import {PerpsMarketFactory} from "../storage/PerpsMarketFactory.sol";
+
+/**
+ * @title LiquidationAssetManagerModule send liquidity to the reward distributor according to each collateral type
+ */
+contract LiquidationAssetManagerModule {
```

Comment:
i think can probably just move this into `CollateralConfigurationModule`.  there won't be a lot in there except for the LAM stuff